.github/workflows/ci.yml

```diff
@@ -26,6 +31,21 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
+      - name: Install system packages
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update && \
+          sudo apt-get -y install \
+            tmux \
+            ;
+
+      - name: Install system packages
+        if: matrix.os == 'macos-latest'
+        run: |
+          brew install \
+            tmux \
+            ;
```

Comment:
Mostly future-proofing for readability. If packages get added/removed in the future, it makes it easy to see right away from the diff which package was changed, since it'll be a one line change and one package per line.

Probably silly but :man_shrugging: